launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/ForkScaffolder.kt

```diff
@@ -29,8 +29,6 @@ class ForkScaffolder(private val launcher: Launcher) {
             instance.local-publish.httpUrl={{instancePublishHttpUrl}}
             instance.local-publish.openPath=/crx/packmgr
 
-            mvnBuild.args={{mvnBuildArgs}}
```

Comment:
why have you removed it?

Reply:
This was by mistake, I'm putting it back